lib/plugins.js

```diff
@@ -389,6 +389,10 @@ exports.categories = {
         tv: {
             url: 'https://github.com/hapijs/tv',
             description: 'Interactive debug console plugin for hapi servers'
+        },
+        'hapi-recaptcha': {
+            url: 'https://github.com/cristobal15/hapi-recaptcha',
+            description: 'Google's reCaptcha for hapi'
```

Comment:
what he means by that is you need to escape the single quote character like `Google\'s`